app/views/withdraw_items/new.html.erb

```diff
@@ -33,7 +33,7 @@
       <%= f.text_area 'comments', class: 'form-control' %>
     </div>
   </div>
-  <%= f.hidden_field :load_date, value: DateTime.now.midnight %>
+  <%= f.hidden_field :load_date, value:  l(Time.now.getlocal, format: :oracle) %>
```

Comment:
Confirmed with Darsi that the load_date doesn't need to be midnight (not sure why we did this in the first place).